.xo-config.js

```diff
@@ -44,7 +44,28 @@ module.exports = {
 		// Temporarily disabled
 		// TODO: Need to decide what to do with this rule. Either keep it disabled here or remove it from XO.
 		'@typescript-eslint/no-invalid-void-type': 'off', // Reason: https://github.com/sindresorhus/refined-github/pull/3168#discussion_r434479434
-		'unicorn/no-fn-reference-in-iterator': 'off'
+		'unicorn/no-fn-reference-in-iterator': 'off',
+		'@typescript-eslint/no-extra-non-null-assertion': 'error',
+		'@typescript-eslint/consistent-type-definitions':'error',
+		'@typescript-eslint/require-await': 'error',
```

Comment:
FYI: https://github.com/xojs/eslint-config-xo-typescript/blob/7842d05deaadff1ced9a46f3e56d786e7b2922d2/index.js#L443

Reply:
I remember this. It looks like the lint is failing because of this one.

If it works painlessly on the current code, we can enable it.

Reply:
This was the only one I was having issues with (hence my first comment)